sdk/eventhub/Azure.Messaging.EventHubs/tests/Consumer/EventHubConsumerClientLiveTests.cs

```diff
@@ -1953,11 +1953,10 @@ await using (var consumer = new EventHubConsumerClient(EventHubConsumerClient.De
         [TestCase(EventHubsTransportType.AmqpWebSockets)]
         public async Task ConsumerCanRetrievePartitionProperties(EventHubsTransportType transportType)
         {
-            var partitionCount = 4;
+            var partitionCount = 1;
```

Comment:
For my own understanding - Why the changes to this file?  My guess is just general cleanup (I see how this test is very similar to the new tests being adder for the PartitionReceiver) and here we don't actually need more than a single partition - but I'm wondering if there was something more subtle going on?  If it's just cleaning up code - Nice Cleanup!

Reply:
It's just cleanup. Like you said, we were using only one partition.